applications/openshift/api-server/api_server_auth_mode_rbac/rule.yml

```diff
@@ -0,0 +1,38 @@
+prodtype: ocp4
+
+title: Must have authorization-mode RBAC set
+
+description: 'Turn on Role Based Access Control'
+
+rationale: |-
+  Role Based Access Control (RBAC) allows fine-grained control over the
+  operations that different entities can perform on different objects in
+  the cluster.
```

Comment:
Cool. Some more sentences on security would be great.

Reply:
added something, can you check?